query-engine/connector-test-kit/src/test/scala/writes/nestedMutations/NestedDisconnectMutationInsideUpdateSpec.scala

```diff
@@ -939,7 +944,8 @@ class NestedDisconnectMutationInsideUpdateSpec extends FlatSpec with Matchers wi
          |}
       """,
       project,
-      errorCode = 3041
+      errorCode = 0, // 3041
+      errorContains = """Error occurred during query execution:\nInterpretationError(\"Error for binding \\'1\\': RecordsNotConnected { relation_name: \\\"UserFollows\\\", parent_name: \\\"User\\\", child_name: \\\"User\\\" }"""
```

Comment:
The old error codes are defined here: https://github.com/prisma/prisma/blob/56ee4251864b782330865e3e8ebdf89a25e9a0da/server/connectors/api-connector/src/main/scala/com/prisma/api/schema/Errors.scala#L103

Should we try to reimplement similar handling for 3041 and 3042?